deploy/charts/cert-manager-crds/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- munnerz
+- simonswine
+- kragniz
+reviewers:
+- munnerz
+- unguiculus
+- kragniz
```

Comment:
Can you remove this file altogether? We don't need a special OWNERS file here really :) this made more sense when we used `helm/charts` to publish the chart.